packages/asset/contracts/Asset.sol

```diff
@@ -370,5 +373,10 @@ contract Asset is
         OperatorFiltererUpgradeable._setOperatorFilterRegistry(registry);
     }
 
+    /// @notice initializes the ownable extension
+    function reinitialize() public reinitializer(2) {
```

Comment:
Rather than hardcoding the `2` can we pass this as a param in the deploy script?

Reply:
Do we also need to remove the code at line 60 ?
```
 /// @custom:oz-upgrades-unsafe-allow constructor
    constructor() {
        _disableInitializers();
    }
```

Reply:
> Or is it actually preferable to keep the `2` hardcoded for clarity regarding what the current version is?

We might hardcode it for clarity, it should always be going +1 when we need to reinitialize again.

Reply:
> Do we also need to remove the code at line 60 ?
> 
> ```
>  /// @custom:oz-upgrades-unsafe-allow constructor
>     constructor() {
>         _disableInitializers();
>     }
> ```

Hmm that is a good point, interestingly it allowed the contract to be reinitialized anyway. 🤔

Reply:
This is to avoid the initialization of the implementation contract, not the proxy.

Reply:
Ah that's correct, yeah, so its safe to keep it